botocore/parsers.py

```diff
@@ -1113,8 +1116,36 @@ def _handle_string(self, shape, text):
         return text
 
 
+class S3Parser(RestXMLParser):
+    def _parse_headers(self, final_parsed, headers, member_shape, name):
+        header_name = member_shape.serialization.get('name', name)
+        if header_name in headers:
+            if header_name == 'Expires':
+                final_parsed[name] = self._handle_expires_timestamp(
+                    member_shape, headers[header_name]
+                )
+                final_parsed['ExpiresString'] = headers[header_name]
```

Comment:
super nit: I'd make this the first line in the conditional for readability. It doesn't rely on the previous statement and is in-between two statements that are related to each other.